### PR TITLE
technical update to Feature on Demand

### PR DESCRIPTION
**Pull request by Karl-WE**
I have added some clarification on Feature on demand. The list "as noted" is not meant to be complete.
myself have tracked recent developements of FoD as an active user of FoD and also Windows Insider and additional to this PR, I would ask you for permission to update the table below, to make clear which FoD package added functionality.

I hope you appreciate my initiative, as now vServer Next is hitting the next LTSC insider release with b20201. 

Currently the list and other docs may appear still unappealing to IT Pros to leverage Server Core, since many restrictions existed based on 1809 do no longer exist as of today technology.

### CODE REVIEW

**Comment by PRMerger12** [comment]
@Karl-WE : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/administration/server-core/what-is-server-core.md [hunk 1]
```diff
@@ -37,7 +37,25 @@ In addition to no UI, Server Core also differs from the Server with Desktop Expe
...
+The following table shows which applications are available *locally* on Server Core vs Server with Desktop Experience. **Important**: In most cases, applications that are listed as "not available" below can be run remotely from a Windows client computer or [Windows Admin Center](../../manage/windows-admin-center/overview.md) to manage your Server Core installation. 
+
+### Versioning and Downloads of Feature on Demand for Server Core
+
+The list below reflects the FoD level as of Version 1809 (Server 2019 LTSC). 
+You can greatly enchance your Server Core compatibility by applying a FoD package, newer than 1809.
+
+*Download locations*
+
+Later Windows Server (SAC) FoD, can be obtained in your specific [MS Licensing Center](https://businessaccount.microsoft.com/), if you have active Software Assurance.
```
Comment (illfated) [inline WindowsServerDocs/administration/server-core/what-is-server-core.md:49]:
```suggestion
Later Windows Server (SAC) FoD can be obtained from your specific [MS Licensing Center](https://businessaccount.microsoft.com/), if you have got active Software Assurance.
```

File: WindowsServerDocs/administration/server-core/what-is-server-core.md [hunk 2]
```diff
@@ -37,7 +37,25 @@ In addition to no UI, Server Core also differs from the Server with Desktop Expe
...
+You'll need a Microsoft Work or School Account enrolled to VLSC or Business Center, to access the FoD download.
```
Comment (illfated) [inline WindowsServerDocs/administration/server-core/what-is-server-core.md:50]:
```suggestion
You'll need a Microsoft Work or School Account enrolled to VLSC or Business Center to access the FoD download.
```

File: WindowsServerDocs/administration/server-core/what-is-server-core.md [hunk 3]
```diff
@@ -37,7 +37,25 @@ In addition to no UI, Server Core also differs from the Server with Desktop Expe
...
+
+Alternatively, you might have access to a [Visual Studio Subscription](https://my.visualstudio.com/).
+You'll need a Microsoft Home, Work or School account, enrolled to an active Visual Studio subscription, to access the FoD download.
+
+Latest Server Insider FoD. This is downloadable at [Server Insider webpage](https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewserver).
```
Comment (illfated) [inline WindowsServerDocs/administration/server-core/what-is-server-core.md:55]:
```suggestion
Latest Server Insider FoD. This is downloadable at the [Server Insider webpage](https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewserver). 
```

**Comment by PRMerger18** [comment]
@Karl-WE : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger18** [comment]
@Karl-WE : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger18** [comment]
@Karl-WE : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Karl-WE** [comment]
Thanks again for corrections! @illfated you are awesome :)

File: WindowsServerDocs/administration/server-core/what-is-server-core.md [hunk 4]
```diff
@@ -37,7 +37,25 @@ In addition to no UI, Server Core also differs from the Server with Desktop Expe
...
 
-The following table shows which applications are available *locally* on Server Core vs Server with Desktop Experience. **Important**: In most cases, applications that are listed as "not available" below can be run remotely from a Windows client computer and used to manage your Server Core installation.
+The following table shows which applications are available *locally* on Server Core vs Server with Desktop Experience. **Important**: In most cases, applications that are listed as "not available" below can be run remotely from a Windows client computer or [Windows Admin Center](../../manage/windows-admin-center/overview.md) to manage your Server Core installation. 
+
+### Versioning and Downloads of Feature on Demand for Server Core
+
+The list below reflects the FoD level as of Version 1809 (Server 2019 LTSC). 
+You can greatly enchance your Server Core compatibility by applying a FoD package, newer than 1809.
+
+*Download locations*
```
Comment (illfated) [inline WindowsServerDocs/administration/server-core/what-is-server-core.md:47]: This line looks like it should have been a subheading for the previous H3 heading, so perhaps use a H4 heading here?
```suggestion
#### Download locations
```

**Comment by PRMerger12** [comment]
@Karl-WE : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by illfated — APPROVED** [review]
Looks good to me. Will defer to final review and approval by the author.

**Comment by Karl-WE** [comment]
hope your fine @pronichkin @lizap. Any comments / suggestions from your end or is it ok to sign-off?

**Pull request merged**